packages/auth/index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+
+<head>
+  <title>OAuth tests</title>
+  <button id="loginBtn">Login</button>
+  <script src="dist/auth.umd.js"></script>
+</head>
+
+<body>
+  <script>
+    const oauth = new cartoToolkitAuth.OAuth({
+      clientID: 'fmka4wCAclcn',
+      redirectURI: 'https://localhost:5000'
+    });
+
+    loginBtn.addEventListener('click', async () => {
+      const data = await oauth.login();
+      console.log('Got data from login promise:', data);
+    });
+
+    oauth.on('tokenUpdated', (d) => {
+      console.log('Token updated', oauth.credentials);
+    });
+
+    oauth.on('error', () => {
```

Comment:
I guess event includes an object with the error cause, right?

Reply:
I just checked, only in one case, I should change it so it always comes